cluster/plan.go

```diff
@@ -126,8 +126,8 @@ func (c *Cluster) BuildKubeAPIProcess(prefixPath string) v3.Process {
 		"client-ca-file":                     pki.GetCertPath(pki.CACertName),
 		"tls-cert-file":                      pki.GetCertPath(pki.KubeAPICertName),
 		"tls-private-key-file":               pki.GetKeyPath(pki.KubeAPICertName),
-		"kubelet-client-certificate":         pki.GetCertPath(pki.KubeAPICertName),
-		"kubelet-client-key":                 pki.GetKeyPath(pki.KubeAPICertName),
+		"kubelet-client-certificate":         pki.GetCertPath(pki.KubeNodeCertName),
```

Comment:
we use kubeapi cert because its a authentication from kubeapi to kubelet, not the other way around